### PR TITLE
added padding to pop up bottom

### PR DESCRIPTION
**Pull request by ritabiniash**
image

### CODE REVIEW

File: packages/coordinator/src/components/Popup/Popup.tsx [hunk 1]
```diff
@@ -51,7 +55,7 @@ export function Popup(props: PopupProps) {
     <Dialog open={open} onClose={onClose} fullWidth={true}>
       <DialogTitle>{title}</DialogTitle>
       <DialogContent classes={classes}>{children}</DialogContent>
-      <DialogActions classes={classes}>
+      <DialogActions className={clsx(classes.root, classes.bottom)}>
```
Comment (yaron1m) [inline packages/coordinator/src/components/Popup/Popup.tsx:58]: We don't have a dependency on `clsx`.

Please use `classNames(...)`

File: packages/coordinator/src/components/Popup/Popup.tsx [hunk 2]
```diff
@@ -43,6 +44,9 @@ export function Popup(props: PopupProps) {
       textAlign: "center",
       paddingBottom: "10px",
     },
+    bottom: {
+      paddingBottom: "30px",
```
Comment (yaron1m) [inline packages/coordinator/src/components/Popup/Popup.tsx:48]: Do we want this in all popups? Or just the delete appointment one?

Comment (ritabiniash) [reply packages/coordinator/src/components/Popup/Popup.tsx:48]: all pf them

File: packages/coordinator/src/components/Popup/Popup.tsx (hunk 1 above)
Comment (ritabiniash) [reply packages/coordinator/src/components/Popup/Popup.tsx:58]: ok will change. it worked locally :O

Comment (yaron1m) [reply packages/coordinator/src/components/Popup/Popup.tsx:58]: I guess we have it as a transitive dependency

File: packages/coordinator/src/components/Popup/Popup.tsx (hunk 2 above)
Comment (yaron1m) [reply packages/coordinator/src/components/Popup/Popup.tsx:48]: Did you check how the other popups look after this change? Do they look OK?

Comment (ritabiniash) [reply packages/coordinator/src/components/Popup/Popup.tsx:48]: Good point :) yes checked them 
BTW we wanted this behavior when we originally wrote the pop-up but there was an issue so it was reverted

Comment (yaron1m) [reply packages/coordinator/src/components/Popup/Popup.tsx:48]: Great.

Yedidya just pushed another usage of the popup, please take a look at it as well, and if it looks good let's merge this :)

**Review by yaron1m — APPROVED** [review]

**Pull request merged**